examples/third_party/BUILD.bazel

```diff
@@ -28,8 +28,8 @@ test_suite(
     tags = ["manual"],
     tests = [
         "//apr:apr_build_test",
-        "//apr_util:apr_util_build_test",
         # Missing a new enough m4 to build
+        # "//apr_util:apr_util_build_test",
```

Comment:
Nit: it's failing to build due to not preserving time stamps rather than a dep on m4 - can you add a comment to this effect?

Reply:
Yep for some reason I mistook “m4” with “autotools”. 

I’m on holiday for a week now but will change the comment when I get back :)

Reply:
done :)